targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/TARGET_MUSCA_A1_NS/prebuilt/README.md

```diff
@@ -0,0 +1,43 @@
+# MUSCA_A1_NS prebuilt secure binaries
+
+This directory tree contains Secure images released under Permissive Binary License.
+
+Built by mbed-cli using GNU Arm Embedded - version 6.3.1
+
+These images were compiled by the following command:
+
+## mcuboot.bin
+### Repository
+https://git.trustedfirmware.org/trusted-firmware-m.git
+### Commit SHA
+8da7f102a6a6a1a99462f7f32edbd1565096c2f3
+```sh
+cmake ../ -G"Unix Makefiles" -DTARGET_PLATFORM=MUSCA_A -DCOMPILER=ARMCLANG -DCMAKE_BUILD_TYPE=Debug
+make
+```
+
+## tfm.bin
+
+```sh
+mbed compile -t GCC_ARM -m ARM_MUSCA_A1_S --app-config tools/psa/tfm/mbed_app.json --profile release
+```
+
+## spm_smoke.bin
```

Comment:
do we need all these test binaries? They add 1.5mb to the clone size, given how mbed-cli works this is not desirable at all.

Reply:
That is how it is being done so far, we shall revisit this in future (not the PRs scope). 

cc @mbed-os-tools FYI